jobs/haproxy/spec

```diff
@@ -50,6 +50,10 @@ properties:
     default: 80
   ha_proxy.additional_frontend_config:
     description: "Configurations which are passed to the http-in and https-in configuration block"
+  ha_proxy.user_agent_frontend:
```

Comment:
From meatspace: we'll rename to `enable_healthcheck_frontend` to describe the functionality rather than the implementation.